wagtail/admin/templates/wagtailadmin/shared/shared_avatar.html

```diff
@@ -0,0 +1,12 @@
+{% load wagtailadmin_tags %}
+<span class="avatar {{size}}" {% if tooltip %} data-wagtail-tooltip="{{ tooltip }}" {% endif %}>
```

Comment:
Let's use the `classname` util here and also make it possible to pass a classname in

```html
<span class='{% classnames "avatar" size classname %}' {% if tooltip %} data-wagtail-tooltip="{{ tooltip }}" {% endif %}>
```